tests/test_relational_operand.py

```diff
@@ -0,0 +1,596 @@
+import pytest
+import random
+import string
+import pandas
+import datetime
+import numpy as np
+import datajoint as dj
+from .schema_simple import (
+    A,
+    B,
+    D,
+    E,
+    F,
+    L,
+    DataA,
+    DataB,
+    TTestUpdate,
+    IJ,
+    JI,
+    ReservedWord,
+    OutfitLaunch,
+)
+from .schema import (
+    Experiment,
+    TTest3,
+    Trial,
+    Ephys,
+    Child,
+    Parent,
+    SubjectA,
+    SessionA,
+    SessionStatusA,
+    SessionDateA,
+)
+from . import PREFIX, CONN_INFO
+
+
+@pytest.fixture
+def schema_simp_pop(schema_simp):
+    """
+    Schema simple with data populated.
+    """
+    og_a_contents = A.contents.copy()
+    og_l_contents = L.contents.copy()
+    B.populate()
+    D.populate()
+    E.populate()
+    yield schema_simp
+    A.contents = og_a_contents
+    L.contents = og_l_contents
+
+
+@pytest.fixture
+def schema_any_pop(schema_any):
+    """
+    Schema any with data populated.
+    """
+    Experiment.populate()
+    yield schema_any
+
+
+def test_populate(schema_simp_pop):
+    assert not B().progress(display=False)[0], "B incompletely populated"
```

Comment:
I did not separate `B()` into a separate fixture in this module. In my opinion, the best practice that balances consistency and readability is:

- If the table instance was being instantiated in a `setup_class` in the old tests (like `cls.subject = Subject()`), then create and use a `subject` fixture instead. In these cases, the original test authors used `self.subject` instead of `Subject()` in the tests.
- If the table instance is being constructed inline, as is the case for `B()` in this module, then the original test authors weren't worried about state that carried over between tests, and we can leave the inline construction.

Will gladly take feedback on this convention (which I'm aware is changing as we go through). I could replace every `B()` with a fixture `b`, but I think this would make the module much less readable, even if it's more consistent with other modules that need fixtures for this.